app/server.js

```diff
@@ -7,7 +7,8 @@ const server = Hapi.server({
 
 const routes = [].concat(
   require('./routes/healthy'),
-  require('./routes/healthz')
+  require('./routes/healthz'),
+  require('./routes/api/eligibility')
```

Comment:
Maybe following our pattern we could move this route under the new auto-eligibility folder on my branch?

Reply:
agree, i'll realign with your changes